packages/portfolio-contract/test/portfolio-open.test.ts

```diff
@@ -100,9 +101,9 @@ test('open portfolio', async t => {
   const localP = makeLocalAccount(orch, {});
   const actual = await openPortfolio(
     orch,
-    { zoeTools, makePortfolioKit, inertSubscriber },
+    { zoeTools, makePortfolioKit, inertSubscriber, contractAddresses },
     seat,
-    undefined,
+    { evmChain: 'Ethereum' },
```

Comment:
use the cap-10 ID for Ethereum.

Reply:
Ah. So it's an external design constraint. Include those links in the code, please. As documentation on a type. Was it already there and I missed it?